packages/components/src/spectrum/View.tsx

```diff
@@ -9,6 +9,13 @@ import { type ColorValue, colorValueStyle } from '../theme/colorUtils';
 
 export type ViewProps = Omit<SpectrumViewProps<6>, 'backgroundColor'> & {
```

Comment:
Should add the `border` colour props that you're redefining to the `Omit` here (like `backgroundColor` is), e.g.
```
export type ViewProps = Omit<SpectrumViewProps<6>, 'backgroundColor' | 'borderColor' | ...>
```